commands/misc.py

```diff
@@ -1,11 +1,12 @@
 import time
 from art import *
-from discord.ext.commands import CommandInvokeError
+from discord.ext.commands import CommandInvokeError, command
 
 from util.helpers import mention
 from discord.ext import commands
 
 
+@command
```

Comment:
I think you're registering the command with same decorator twice. 
Should it be imported from `from annotation.futebot import command`?